demo/loadFeatures.py

```diff
@@ -1,48 +1,55 @@
 import json
 import requests
-from utils import ES_AUTH, ES_HOST
+from utils import ES_AUTH, ES_HOST, BASEPATH_FEATURES, FEATURE_SET_NAME
 from urllib.parse import urljoin
 
 
-def getFeature(ftrId):
-    return json.loads(open('%s.json' % ftrId).read())
+# Each feature is stored in a file with the name of the file as feature_number.json: 1.json, 2.json. This function
+# load the file from the filesystem using the basepath from the configuration file.
+def get_feature(ftr_id: int):
+    return json.loads(open(BASEPATH_FEATURES + '%s.json' % ftr_id).read())
 
-def eachFeature():
+
+# Find all available features using the pattern 1.json, 2.json, etc. All features have to be in order as we stop the
```

Comment:
Thanks for the comments - should these be doc strings instead?

Reply:
Good point, I forgot about those, will move the comments to the right location.